src/style.css

```diff
@@ -0,0 +1,111 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir1;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  margin-top: 0;
+}
+
+.form {
+  height: 70px;
+  position: relative;
+  margin-top: 20px;
+  margin-bottom: 20px;
```

Comment:
you can use margin shorthand for this

Reply:
Agree, lets' combine them into 1 property